pkgs/applications/graphics/dosage/default.nix

```diff
@@ -1,28 +1,26 @@
-{ lib, python3Packages, fetchFromGitHub }:
+{ lib, python3Packages }:
 
 python3Packages.buildPythonApplication rec {
   pname = "dosage";
-  version = "2018.04.08";
-  PBR_VERSION = version;
+  version = "2.17";
 
-  src = fetchFromGitHub {
-    owner = "webcomics";
-    repo = "dosage";
-    rev = "b2fdc13feb65b93762928f7e99bac7b1b7b31591";
-    sha256 = "1p6vllqaf9s6crj47xqp97hkglch1kd4y8y4lxvzx3g2shhhk9hh";
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0vmxgn9wd3j80hp4gr5iq06jrl4gryz5zgfdd2ah30d12sfcfig0";
   };
-  checkInputs = with python3Packages; [ pytest responses ];
-  propagatedBuildInputs = with python3Packages; [ colorama lxml requests pbr setuptools ];
 
-  disabled = python3Packages.pythonOlder "3.3";
+  checkInputs = with python3Packages; [ pytestCheckHook ];
```

Comment:
```suggestion
  nativeBuildInputs = [ setuptools-scm ];

  checkInputs = with python3Packages; [ pytestCheckHook ];
```
build fails

Reply:
Hmm, a nix-shell worked for me, I guess because of impurity.
How did you figure setuptools-scm is the missing dependency? A --pure nix-build errors on 'xdist' for me.

Reply:
`python3Packages.setuptools-scm` didn't help with the test failures for me so I added the test dependencies again.

Reply:
> How did you figure setuptools-scm is the missing dependency? A --pure nix-build errors on 'xdist' for me.

python is none deterministic :)